error.js

```diff
@@ -66,9 +67,16 @@ function writeError(req, res, opts, callback) {
     }
 
     if (opts.verbose) {
```

Comment:
We want to make a breaking change and ensure that by default, when not configured, `stack` is not serialized.